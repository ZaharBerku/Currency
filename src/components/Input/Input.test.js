/* eslint-disable no-undef */
import React from "react";
import { Input } from "./index";
import { render } from "@testing-library/react";

const Component = (props) => {
    return(
        <Input type="text" value="input" additionalClassNames="active" placeholder="text" {...props}/>
    );
};


describe("Input snapshot render", () => {
    test("should Input render", () => {
        const { asFragment } = render(<Component/>);
        expect(asFragment()).toMatchSnapshot();
    });
});
