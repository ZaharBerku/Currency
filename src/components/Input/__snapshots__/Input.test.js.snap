// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input snapshot render should Input render 1`] = `
<DocumentFragment>
  <input
    class="active"
    data-testid="input"
    disabled=""
    placeholder="text"
    type="text"
    value="input"
  />
</DocumentFragment>
`;
